src/containers/AppRoot.tsx

```diff
@@ -121,11 +124,20 @@ class AppRoot extends React.Component<Props> {
   }
 
   async promptAndSetDecryptedIdentity(): Promise<void> {
-    console.info('[ENCRYPTION] Decrypting and setting identity in store')
-    const { setIdentityInStore } = this.props
-    const identityDecrypted = await promptUserAndGetIdentityDecrypted()
-    setIdentityInStore(identityDecrypted)
-    console.info('[ENCRYPTION] OK identity set in store')
```

Comment:
Is this because of the `async` nature of the identity now?

Reply:
no, it just didn't work, at least on my device. that fixes the finger scan popup constantly reappearing (see ticket)

Reply:
I didn't make it async or anything, I'm just avoiding that the trigger fires twice in quick succession and blocks everything